src/modal/header/PeraWalletModalHeader.ts

```diff
@@ -24,7 +25,11 @@ peraWalletModalHeader.innerHTML = `
           : `<div class="pera-wallet-modal-header__brand">
               <img src="${PeraConnectIcon}" />
 
-              Pera Connect
+              <div class="pera-wallet-modal-header__brand-text">
+                <span>Pera Connect</span>
+
+                <span class="pera-wallet-modal-header__version-number">${PERA_CONNECT_VERSION_NUMBER}</span>
```

Comment:
Importing the package.json directly increasing the bundle size by ~8kb, let's pass the variable from Rollup.

We can use `@rollup/plugin-replace`, which is basically replaces a string with another string.

Let's add this to the end of the `plugins` section of the Rollup config
```typescript
replace({
  PERA_CONNECT_VERSION: `v${PeraConnectVersion}`,
  preventAssignment: true
})

```

And use it directly as string, `PERA_CONNECT_VERSION` string will be replaced by `v1.2.3` on build time
```suggestion
                <span class="pera-wallet-modal-header__version-number">PERA_CONNECT_VERSION</span>
```